tests/output/ConsoleInteractionTest.py

```diff
@@ -938,6 +939,12 @@ def test_show_bears_empty(self):
             show_bears({}, {}, True, True, self.console_printer)
             self.assertIn('No bears to show.', stdout.getvalue())
 
+    def test_show_bears_with_json(self):
+        args = default_arg_parser().parse_args(['--json'])
+        with retrieve_stdout() as stdout:
+            show_bears({}, {}, True, True, self.console_printer, args)
+            self.assertIn('[]', stdout.getvalue())
```

Comment:
How come your test is passing with this change I suggested in earlier comment without any code changes ?
And actual output I'm getting on my local is:
```
{
  "bears": []
}
```

Reply:
https://github.com/coala/coala/pull/4877#issuecomment-347036274

Reply:
Oh sorry, my bad. I'll try to fix this :)